build_docs.py

```diff
@@ -295,6 +295,7 @@ def __gt__(self, other):
     Language("zh-cn", "zh_CN", "Simplified Chinese", True, XELATEX_WITH_CJK),
     Language("zh-tw", "zh_TW", "Traditional Chinese", True, XELATEX_WITH_CJK),
     Language("pl", "pl", "Polish", False, XELATEX_DEFAULT),
+    Language("uk", "uk", "Ukrainian", True, XELATEX_DEFAULT),
```

Comment:
```suggestion
    Language("uk", "uk", "Ukrainian", False, XELATEX_DEFAULT),
```
`in_prod` is responsible for displaying in language switcher, which according to https://peps.python.org/pep-0545/#add-support-for-translations-in-docsbuild-scripts should be done after having those priority resources translated.

Reply:
Thanks for your suggestions.
These documents are already translated and are reviewing now.

Reply:
Wow, congrats! Wasn't expecting that :)